🌐 Add German translation for `docs/de/docs/tutorial/security/get-current-user.md`

nilslindemann opened the pull request:
← `tutorial/security/first-steps.md` (#10432)
→ `tutorial/security/simple-oauth2.md` (#10504)

[German translation progress](https://github.com/tiangolo/fastapi/discussions/10582)

Commit:
🌐 Add German translation for `docs/de/docs/tutorial/security/get-current-user.md`

* Full translation
* In sync with current docs/en/docs/tutorial/security/get-current-user.md
* Google Translate used
* Manually checked
* Checked for errors using LanguageTool

tiangolo commented:
📝 Docs preview for commit df2b553556846400eb452d67b185a8eff4b6413d at: https://59c10ddc.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/tutorial/security/get-current-user.md

tiangolo commented:
📝 Docs preview for commit 4178fd3a94427da62b8b18853dda83ed0b5d9fae at: https://3ee79c61.fastapitiangolo.pages.dev

Commit:
Sync with #10442

Commit:
Merge branch 'tiangolo:master' into de/docs/tutorial/security/get-current-user.md

tiangolo commented:
📝 Docs preview for commit 7ab9f4d226a7dc4e2fb51a5c552a91513aa4c8e8 at: https://5f816921.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/tutorial/security/get-current-user.md

tiangolo commented:
📝 Docs preview for commit 1637d4f37eaa69377a4553e092652ac0e1b605bb at: https://7c072d40.fastapitiangolo.pages.dev

Commit:
"Tip" -> "Tipp"

tiangolo commented:
📝 Docs preview for commit e7c69d75d63c91b0f0366da718f216069ce70544 at: https://d1744c5b.fastapitiangolo.pages.dev

Commit:
"Das Token" -> "Der Token"

Sources:
https://www.korrekturen.de/genus/der-oder-das-token.shtml
https://rechtschreibung.woxikon.de/artikel/der-oder-das-token

tiangolo commented:
📝 Docs preview for commit 4e6e1fa3dda2a0f6f26b01530ed7ae984be8a988 at: https://1e2d3af3.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/tutorial/security/get-current-user.md

tiangolo commented:
📝 Docs preview for commit 20b20d5c5b40d00f29ca8144a7c9f866b62e5b34 at: https://d58e91b5.fastapitiangolo.pages.dev

Commit:
"Nutzer" -> "Benutzer"

tiangolo commented:
📝 Docs preview for commit 93e60d9736ae7fe7e32b67fc164b78c7210eed10 at: https://53258229.fastapitiangolo.pages.dev

Commit:
Use typographic quotes

tiangolo commented:
📝 Docs preview for commit 93e34cd17428d9928221b23eeb3676c1f4a2d451 at: https://a7cde4f3.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/tutorial/security/get-current-user.md

tiangolo commented:
📝 Docs preview for commit 1febc2633485b12173bc9517a5711c580bdca451 at: https://31edaca6.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/tutorial/security/get-current-user.md

tiangolo commented:
📝 Docs preview for commit b3703f65ce198e0aa0f176f2c6b872d3aab1aced at: https://e826bc8d.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/tutorial/security/get-current-user.md

Commit:
Merge branch 'tiangolo:master' into de/docs/tutorial/security/get-current-user.md

tiangolo commented:
📝 Docs preview for commit 2851efe8c533c878387ff97ce5de3d0b7979aacc at: https://75f62e3b.fastapitiangolo.pages.dev

Commit:
"-Modell" -> "modell"

Commit:
Merge branch 'tiangolo:master' into de/docs/tutorial/security/get-current-user.md

tiangolo commented:
📝 Docs preview for commit dd5457599b65fbadb04fa65da5f2a44174451b03 at: https://861226bf.fastapitiangolo.pages.dev

tiangolo commented:
Awesome @nilslindemann! :rocket: 

As you are the champion of German translations and you have done so many, and with so much attention to detail, I'm gonna go ahead and merge this. :tada: 

If others see things that could be improved later, we can check that in new PRs. :nerd_face:
